javascript/chronometer.js

```diff
@@ -1,26 +1,65 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+   this.currentTime = 0;
+   let intervalId = setInterval(() =>{
```

Comment:
you are saving the interval in a simple variable but you are not updating the class property, for that you need this.intervalId. The reason you do this is to later be able to clear it.